operators/src/processing/reprojection.rs

```diff
@@ -126,7 +128,7 @@ impl InitializedOperator<VectorResultDescriptor, TypedVectorQueryProcessor>
     }
 }
 
-struct ReprojectionProcessor<Q, G>
+struct VectorReprojectionProcessor<Q, G>
```

Comment:
This seems to be a good idea. But where is the `RasterReprojectionProcessor`?

Reply:
at that time there was none

Reply:
was added in 89a884e . However, there are many changes following that commit.